scripts/Dockerfile

```diff
@@ -2,7 +2,9 @@ FROM ubuntu:latest
 
```

Comment:
This dockerfile is also be used in releasing images, shall we put in other files?

Reply:
I don't get the point. What does "other files" mean here?

Reply:
I got it mixed up, resolved.